repositories.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE repositories SYSTEM "/dtd/repositories.dtd">
+<repositories xmlns="" version="1.0">
+  <repo priority="50" quality="experimental" status="unofficial">
+    <name><![CDATA[ml-overlay]]></name>
```

Comment:
why CDATA here and there ?

Reply:
Do not know, I just copied repositories.xml from flatpak-overlay :)

Reply:
then please remove it: https://api.gentoo.org/overlays/repositories.xml
does not use that